dockerfiles/theia/README.md

```diff
@@ -1,33 +1,101 @@
 # How to Build Theia Image
 
 ## Build image manually using build scripts
-To build Theia image manually You could use build.sh scripts.
-Dockerfile uses multi-stage build feature, so first of all You need build builder image -
-theia-dev:
 
-Example:
+To build all the images you can easily run [build.sh](../../build.sh) script located in the repository root.
```

Comment:
so this is not the `repository root` but the dockerfiles folder ... no ?

Reply:
In the repository root

Reply:
ok i think it is very confusing because we have build.sh in every level. So maybe add a note and which is for what

Reply:
I created an [issue](https://github.com/eclipse/che/issues/14094) on that improvement.